src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+
+class ErrorMessages {
+  static TITLE = 'We are currently having some issues with VM portal.'
+  static DESCRIPTION = 'Please refresh page or log out and log back in. If the issue persists, please report a bug on GitHub: '
+}
+
+class GlobalErrorBoundary extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { hasError: false }
+  }
+
+  componentDidCatch (error, info) {
```

Comment:
Have you looked at `info` and what is available in dev and prod compiles?

It is supposed to be "component stack" and could be used to locate the exact component in the App where the error came from.  That may be useful.

Reply:
> It is supposed to be "component stack" and could be used to locate the exact component in the App where the error came from. That may be useful.
+1 Agree, it would be nice to return not full stack, but error, and name of component that cause error.